sabre_smart_permission_examples/README.md

```diff
@@ -0,0 +1,22 @@
+# Smart Permission Examples
+
+Serveral examples of smart permissions using the sawtooth sabre sdk.
```

Comment:
Serveral  -> Several

Reply:
Better yet, drop "Serveral" and start with "Examples".